static/src/javascripts-legacy/bootstraps/commercial.js

```diff
@@ -72,15 +72,12 @@ define([
     var secondaryModules = [
         ['cm-stickyTopBanner', stickyTopBanner.init],
         ['cm-fill-advert-slots', fillAdvertSlots.init, fillAdvertSlots.customTiming],
-        ['cm-paidContainers', paidContainers.init]
+        ['cm-paidContainers', paidContainers.init],
+        ['cm-paidforBand', paidforBand.init]
     ];
 
     var customTimingModules = [];
 
-    if (config.page.isAdvertisementFeature) {
-        secondaryModules.push(['cm-paidforBand', paidforBand.init]);
```

Comment:
Out of interest, what is the reasoning behind remove the check and adding `cm-paidforBand` straight to secondary modules?